crates/newtype-uuid/src/lib.rs

```diff
@@ -329,6 +329,12 @@ impl<T: TypedUuidKind> FromStr for TypedUuid<T> {
     }
 }
 
+impl<T: TypedUuidKind> Default for TypedUuid<T> {
+    fn default() -> Self {
```

Comment:
Could you mark this `#[inline]` since this is a trivial method?

Reply:
done! see 87cd63a3